provision/testdata/flavor_openshift_46_baremetal.apic.txt

```diff
@@ -0,0 +1,1867 @@
+/api/mo/uni/infra/vlanns-[kube-pool]-static.json
```

Comment:
Currently the infraRsFuncToEpg is not being configured at all for node EPG. Please revisit and validate this file again after the "anchor1" option is removed from this flavor's options per suggestion above.

Reply:
just noticed that acirc is being checked in, I hope it's by mistake.